src/App.js

```diff
@@ -1,15 +1,90 @@
 import SearchBar from './components/SearchBar/SearchBar';
 import Users from './components/Users/Users';
 import './App.css';
+import axios from 'axios';
+import { useEffect, useState } from 'react';
 
 function App() {
+  // hooks
+  const [users, setUsers] = useState([]);
+  const [searchInput, setSearchInput] = useState('');
+  const [expanded, setExpanded] = useState([]);
+  const [loading, setLoading] = useState(true);
+  const [error, setError] = useState('');
+
   // TODO: Fetch data here
+  useEffect(() => {
+    axios.get('https://users-app-backend.onrender.com/users')
+      .then((response) => {
+        setUsers(response.data.data);
+        if (response){
+          setLoading(false);
+        }
+      })
+      .catch((error) => {
+        setLoading(true);
+        setError("Error fetching data from API");
+      })
+  }, []);
+
+  // filter the users that match the `searchInput`
+  // the users data is an object
+  let filteredData = users;
+
+  if (searchInput){
+    filteredData = users.filter(user => {
+    const validInput = `${user.name} ${user.country} ${user.company}`;
+    const validInputToLowerCase = validInput.toLowerCase();
+    const searchInputToLowerCase = searchInput.toLowerCase();
+    return validInputToLowerCase.includes(searchInputToLowerCase);
```

Comment:
Nice! This is an efficient approach because you only lowercase `searchInput` once.